rules.py

```diff
@@ -1086,15 +1086,20 @@ def SetupBuildEnvironment(conf):
         # Disable MSVC paranoid warnings
         env.Append(CCFLAGS = ['/D_SCL_SECURE_NO_WARNINGS', '/D_CRT_SECURE_NO_WARNINGS'])
         # Stop Windows.h from including a lot of useless header files
-        env.Append(CCFLAGS = ['/DWIN32_LEAN_AND_MEAN'])
+        env.Append(CCFLAGS = '/DWIN32_LEAN_AND_MEAN')
+        # Enable full symbolic debugging information
+        env.Append(CCFLAGS = '/Z7')
```

Comment:
Those flags are used to build agent and extension. Am I right ? 
If this is true, do we have to add those flags to extension ?

Reply:
Nope, just stuff that's build directly by SCons, so agent and utils, not extension.